### PR TITLE
Use nAVAX units in get-balance 

### PR DESCRIPTION
**Pull request by matthewbauer**
Uses nAVAX to display all units in get-balance operations. Update test to correctly handle this.

### CODE REVIEW

File: tests/basic-tests.bats [hunk 1]
```diff
@@ -28,34 +28,34 @@ transfer(){
...
   # echo getBalance
 
   run getBalance
   [ "$status" -eq 0 ]
-  [[ "$(echo "$output" | tail -n1 | awk '{print $NF}')" == "371000000" ]]
+  [[ "$(echo "$output" | tail -n1)" == "371000000 nAVAX" ]]
 
   # Transfer the rest away so we are clean for other tests
-  run transfer 370000000nAVAX $FAKE_USER
+  run transfer "370000000 nAVAX" $FAKE_USER
```
Comment (jonored) [inline tests/basic-tests.bats:54]: For what it's worth, the "run" and [ "$status" -eq 0 ] here are redundant; the tests run with -e set, so just running the command will cause the test to fail if it fails. You only need run to test nonzero return values and/or output in a variable.

**Review by ApolloUnicorn — APPROVED** [review]
lgtm

File: tests/basic-tests.bats (hunk 1 above)
Comment (jonored) [reply tests/basic-tests.bats:54]: You also get the output to stdout on a failure case if you don't use run, which is useful for debugging.

```
sometest
```
and
```
run sometest
echo "$output"
[ "$status" -eq 0 ]
```
are pretty much the same thing.

**Review by jonored — APPROVED** [review]
LGTM; we probably want to loop back and show in units of AVAX at some point, but this solves the issue.

**Pull request merged**